Italian Translations Setup (#708)
italia/eid-wallet-it-docs: docs/it/functionalities.rst

    :width: 100%

    Fasi dell’Esperienza Utente di utilizzo di una Istanza IT-Wallet 

Le sezioni a seguire si focalizzano sulle macro-fasi di utilizzo e post-utilizzo. Esse definiscono i requisiti funzionali a supporto dell’Esperienza Utente relativi alle fasi di attivazione ottenimento, presentazione, gestione e disattivazione, unitamente ai requisiti di interazione con il servizio in termini di gestione degli errori, richiesta di assistenza e raccolta feedback. 

Le Risorse Ufficiali descrivono le modalità di interazione Utente - Istanza IT-Wallet e le buone pratiche di progettazione al fine di promuovere coerenza tra le diverse Soluzioni IT-Wallet, in termini di modalità di fruizione delle funzionalità. 

Attivazione dell’Istanza IT-Wallet 
----------------------------------

L'attivazione è il passaggio necessario per abilitare l'Utente all'utilizzo delle funzionalità della Soluzione IT-Wallet per l’ottenimento, la presentazione e la gestione dei propri Attestati Elettronici in modo sicuro. Il processo di attivazione consiste in un’operazione di Autenticazione dell'Utente sull’Istanza IT-Wallet tramite la propria identità digitale che consente la generazione del PID.  

> Di seguito i requisiti funzionali e di Esperienza Utente che il Wallet Provider DEVE garantire attraverso la propria Soluzione IT-Wallet: 

Reviewer: ```suggestion
Di seguito i momenti caratterizzanti l'Esperienza Utente nell'utilizzo della Soluzione IT-Wallet: 
```

Pare che di seguito non si leggano requisiti funzionali, ma una user journey

Reviewer 2: È importante che sia dato il requisito normativo al WP di realizzare il prodotto in modo che l'utente possa fare le azioni descritte

```suggestion
Di seguito i requisiti di Esperienza Utente che il Wallet Provider DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```
Se sei d'accordo, aggiorniamo nel resto del documento.

- La rappresentazione del PID DEVE aderire a un insieme definito di specifiche fornite dal Fornitore PID per garantire riconoscibilità, coerenza e omogeneità tra diverse Soluzioni Wallet.

Il Fornitore PID DEVE:

- Implementare un nome/convenzione di denominazione per riferirsi al PID, per garantire coerenza tra tutte le Soluzioni Wallet;
- Definire un insieme chiaro di specifiche per il PID per garantire l'identificazione e la rappresentazione coerenti del PID tra diverse Soluzioni Wallet, in termini di format, struttura e standard di aspetto (ad esempio colore, immagine di sfondo, ecc.).

Ottenimento degli Attestati Elettronici di Attributi 
----------------------------------------------------

Ad attivazione conclusa, l'Utente PUÒ ottenere uno o più Attestati Elettronici di Attributi all'interno della sua Istanza IT-Wallet. 

A seconda delle specifiche esigenze dell’Utente, della tipologia di Attestato Elettronico di Attributi e delle disponibilità offerte dal Fornitore di Soluzione IT-Wallet, dal Fornitore di Attestati Elettronici di Attributi e dal Titolare di Fonte Autentica, l’ottenimento degli Attestati Elettronici di Attributi può avvenire attraverso due modalità: 

> - **dal Catalogo dell’Istanza IT-Wallet**: l’Utente esplora l’elenco degli Attestati Elettronici di Attributi messi a disposizione dalla Soluzione IT-Wallet, seleziona quello di suo interesse e avvia la procedura di richiesta che si conclude con l’ottenimento dell’Attestato Elettronico di Attributi nell’Istanza IT-Wallet; 

Reviewer: ```suggestion
- **dal Catalogo dell’Istanza IT-Wallet**: l’Utente esplora l’elenco degli Attestati Elettronici di Attributi messi a disposizione dalla Soluzione IT-Wallet, seleziona quello di suo interesse e avvia la procedura di richiesta che si conclude con l’ottenimento dell’Attestato Elettronico di Attributi nell’Istanza IT-Wallet (vedi :ref:`Digital Credentials Catalogue <registry-catalogue.rst>`); 
```

> - **da un Touchpoint del Titolare di Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi qualora coincida con il Titolare di Fonte Autentica): l’Utente interagisce col servizio digitale del Titolare di Fonte Autentica che gli permette di ottenere nella sua Istanza IT-Wallet uno specifico Attestato Elettronico di Attributi tramite un Pulsante di Ingaggio.  

Reviewer: ```suggestion
- **da un Touchpoint del Titolare di Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi qualora coincida con il Titolare di Fonte Autentica): l’Utente interagisce col servizio digitale del Titolare di Fonte Autentica che gli permette di ottenere nella sua Istanza IT-Wallet uno specifico Attestato Elettronico di Attributi tramite un Pulsante di Ingaggio (vedi :ref:`Credential Offer Endpoint <credential-issuance-endpoint.rst>`).  
```

qui mettere un riferimento al flusso di credential offer, definito nella sezione pid/eaa issuance

Reviewer 2: Reputi utile inserire questo riferimento specifico oppure è meglio mettere il riferimento a fondo sezione?

> Nonostante le modalità di avvio della richiesta siano diverse, i flussi di ottenimento condividono una struttura e un processo simili. Di seguito, sono illustrati i requisiti funzionali a supporto dell’Esperienza Utente del flusso di ottenimento di un Attestato Elettronico di Attributi dal Catalogo che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 

Reviewer: ```suggestion
Nonostante le modalità di avvio della richiesta siano diverse, i flussi di ottenimento condividono una struttura e un processo simili. Di seguito, sono illustrati i momenti caratterizzanti l’Esperienza Utente durante la fase di ottenimento di un Attestato Elettronico di Attributi dal Catalogo: 
```

Reviewer 2: ```suggestion
Nonostante le modalità di avvio della richiesta siano diverse, i flussi di ottenimento condividono una struttura e un processo simili. Di seguito, sono illustrati i requisiti dell’Esperienza Utente del flusso di ottenimento di un Attestato Elettronico di Attributi dal Catalogo che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```

- **Presentazione da remoto**: l'Utente presenta il PID e/o un set di Attributi contenuti in uno o più Attestati Elettronici tramite l'Istanza IT-Wallet ad un Verificatore di Attestati Elettronici predisposto per la verifica online al fine, ad esempio, di Autenticarsi e fruire dei servizi erogati. 

Presentazione in prossimità 
^^^^^^^^^^^^^^^^^^^^^^^^^^^

La presentazione in prossimità consente all'Utente di esibire il PID e/o un set di Attributi contenuti in uno o più Attestati Elettronici tramite la propria Istanza IT-Wallet, secondo due diverse modalità: 

- **Modalità supervisionata**: l'Utente, tramite l’Istanza IT-Wallet, presenta il PID e/o uno o più Attestati Elettronici di Attributi, a un Verificatore di Attestati Elettronici dotato di un’apposita app o sistema di verifica (e.g. agente delle forze dell’ordine, farmacista, etc.); 

- **Modalità non supervisionata**: l’Utente, tramite l’Istanza IT-Wallet, presenta il PID e/o un set di Attributi contenuti in uno o più Attestati Elettronici a un dispositivo preposto (e.g. tornello, ATM, etc.). 

> Di seguito i requisiti funzionali a supporto dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 

Reviewer: ```suggestion
Di seguito i momenti caratterizzanti l’Esperienza Utente relativi a entrambe le modalità di presentazione utilizzando una Soluzione IT-Wallet. 
```

Reviewer 2: ```suggestion
Di seguito i requisiti dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 
```

- L’Utente autorizza l’operazione utilizzando la modalità di sblocco precedentemente impostata; 
- L’Utente visualizza l’esito positivo della presentazione avvenuta. 

In caso di errori nell’utilizzo dell’Istanza IT-Wallet, il Fornitore di Soluzione IT-Wallet DEVE garantire all’Utente la visualizzazione di messaggi coerenti che lo informino e guidino alla loro risoluzione. Per approfondimenti si rimanda alla sezione :ref:`functionalities:Gestione degli errori`. 

Presentazione da remoto 
^^^^^^^^^^^^^^^^^^^^^^^

La presentazione da remoto consente all'Utente di esibire il PID e/o un set di Attributi contenuti in uno o più Attestati Elettronici, facendo interagire la propria Istanza IT-Wallet con il Touchpoint di un Verificatore di Attestati Elettronici, tramite un apposito Pulsante di Ingaggio.  

Tale presentazione può avvenire in due diverse modalità, sulla base del tipo di dispositivo utilizzato per accedere al servizio di interesse: 

- **Stesso dispositivo**: nel caso in cui l’Utente voglia accedere a un servizio digitale online utilizzando lo stesso dispositivo su cui ha installato l’Istanza IT-Wallet; 

> - **Cross-dispositivo**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 

Reviewer: ```suggestion
- **Altro dispositivo**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 
```

Reviewer 3: Proponiamo di sostituire con versione inglese

```suggestion
- **Cross-device**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 
```

> Di seguito i requisiti funzionali a supporto dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 

Reviewer: ```suggestion
Di seguito i momenti caratterizzanti l’Esperienza Utente e relativi a entrambe le modalità. 
```

Reviewer 2: ```suggestion
Di seguito i requisiti dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 
```

- L’Utente accede alla propria Istanza IT-Wallet utilizzando la modalità di sblocco precedentemente impostata; 
- L’Utente prende visione dei dati del PID e/o gli Attributi richiesti per la presentazione, del nome del Verificatore di Attestati Elettronici che li richiede e delle relative eventuali informative. L’Utente sceglie se presentare o meno eventuali dati del PID e/o Attributi non obbligatori ai fini della presentazione (Divulgazione Selettiva). L’Utente dà il proprio consenso per poter proseguire oppure annulla l’operazione; 
- L’Utente autorizza l’operazione utilizzando la modalità di sblocco precedentemente impostata; 
- L’Utente visualizza nell’Istanza IT-Wallet l’esito positivo della presentazione avvenuta; 
- L’Utente torna al flusso nel Touchpoint del Verificatore di Attestati Elettronici su cui DEVE visualizzare l’esito della presentazione completata. 

In caso di errori nell’utilizzo dell’Istanza IT-Wallet, il Fornitore di Soluzione IT-Wallet DEVE garantire all’Utente la visualizzazione di messaggi coerenti che lo informino e guidino alla loro risoluzione. Per approfondimenti si rimanda alla sezione :ref:`functionalities:Gestione degli errori`. 

> **Cross-dispositivo** 

Reviewer: ```suggestion
**Altro dispositivo** 
```

Reviewer 3: Proponiamo di utilizzare la dicitura in inglese (vedi altri commenti)

```suggestion
**Cross-device** 
```

- L’Utente prende visione dei dati del PID e/o degli Attributi richiesti per la presentazione, del nome del Verificatore di Attestati Elettronici che li richiede e delle relative eventuali informative. L’Utente sceglie se presentare o meno eventuali dati del PID e/o Attributi non obbligatori ai fini della presentazione (Divulgazione Selettiva). L’Utente dà il proprio consenso per poter proseguire oppure annulla l’operazione; 
- L’Utente autorizza l’operazione utilizzando la modalità di sblocco precedentemente impostata; 
- L’Utente visualizza nell’Istanza IT-Wallet l’esito positivo della presentazione avvenuta; 
- L'Utente torna sul Touchpoint del Verificatore di Attestati Elettronici e visualizza l'esito della presentazione completata. 

In caso di errori nell’utilizzo dell’Istanza IT-Wallet, il Fornitore di Soluzione IT-Wallet DEVE garantire all’Utente la visualizzazione di messaggi coerenti che lo informino e guidino alla loro risoluzione. Per approfondimenti si rimanda alla sezione :ref:`functionalities:Gestione degli errori`. 

Autenticazione 
""""""""""""""

> L'Autenticazione è un caso d’uso specifico della presentazione remota e consente all’Utente di accedere in modo sicuro ai servizi resi disponibili da Verificatori di Attestati Elettronici pubblici e privati, presentando il PID ed eventualmente un set di Attributi contenuti negli Attestati Elettronici di Attributi ottenuti. Questo garantisce all’Utente il pieno controllo sui propri dati e la possibilità di condividere anche i soli dati strettamente necessari alla verifica da parte del Verificatore di Attestati Elettronici garantendo a quest’ultimo l’affidabilità, l’autenticità e la validità dei dati presentati. 

Reviewer: ```suggestion
L'Autenticazione è un caso d’uso specifico della presentazione remota e consente all’Utente di accedere in modo sicuro ai servizi resi disponibili da Verificatori di Attestati Elettronici pubblici e privati, presentando il PID ed eventualmente un set di Attributi contenuti negli Attestati Elettronici di Attributi ottenuti. Questo garantisce all’Utente il pieno controllo sui propri dati e la possibilità di condividere anche i soli dati strettamente necessari alla verifica da parte del Verificatore di Attestati Elettronici. 
```

Il processo di Autenticazione può avvenire in entrambe le modalità Stesso dispositivo e cross-device descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo).

Infatti, a livello di Esperienza Utente, il processo di Autenticazione si distingue da un generico flusso di presentazione principalmente per le modalità di avvio del processo, in questo caso reso possibile a partire da uno specifico pulsante, il Pulsante di Autenticazione. 

Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito e DOVREBBE utilizzare le Risorse Ufficiali open source. 

> Qualora i Verificatori di Attestati Elettronici non intendano utilizzare tali risorse open source, POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione ma DEVONO rispettare i requisiti sotto dettagliati, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].   

Reviewer: ```suggestion
Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].

I Verificatori di Attestati Elettronici DOVREBBERO utilizzare le Risorse Ufficiali open source. Qualora non intendano utilizzare tali risorse open source, i Verificatori di Attestati Elettronici POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione.
```

> Il processo di Autenticazione può avvenire in entrambe le modalità stesso dispositivo e cross-dispositivo descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo). 

Reviewer: ```suggestion
Il processo di Autenticazione può avvenire in entrambe le modalità Stesso dispositivo e Altro dispositivo descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo). 
```

siamo sicuri di italianizzare Stesso dispositivo e Altro dispositivo?

io lascerei same e cross device

Reviewer 2: Concordiamo sull'usare la versione inglese. Ritieni utile aggiungerli al glossario?

Author: ```suggestion
Il processo di Autenticazione può avvenire in entrambe le modalità Stesso dispositivo e cross-device descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo).
```

> Qualora i Verificatori di Attestati Elettronici non intendano utilizzare tali risorse open source, POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione ma DEVONO rispettare i requisiti sotto dettagliati, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].   

Reviewer: ```suggestion
I Verificatori di Attestati Elettronici DEVONO rispettare i requisiti sotto dettagliati, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN]. Qualora i Verificatori di Attestati Elettronici non intendano utilizzare tali risorse open source, questi POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione.   
```

Reviewer: quando normiamo, usiamo sempre e chiaramente il soggetto al quale il linguaggio normativo si applica

Reviewer 2: Suggeriamo di integrare la modifica nel paragrafo precedente:

Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito, **unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].** 
I Verificatori di Attestati Elettronici DOVREBBERO utilizzare le Risorse Ufficiali open source. Qualora non intendano utilizzare tali risorse open source, questi POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione.

 
I Verificatori di Attestati Elettronici, in ogni caso, DEVONO abilitare il processo di Autenticazione rendendo disponibili le seguenti pagine: 

- **Discovery Page**: ha l’obiettivo di mostrare all’Utente tutti i metodi di Autenticazione disponibili; 
- **QR code page** (*solo per modalità cross-dispositivo*): ha lo scopo di invitare l’Utente a inquadrare il codice QR;  

> - **waiting page** (*solo per modalità cross-dispositivo*): ha lo scopo di invitare l’Utente a continuare il processo di Autenticazione sulla propria Istanza IT-Wallet; 

Reviewer: rivediamo questi termini cross dispositivo e stesso dispositivo

Reviewer 3: ```suggestion
- **QR code page** (*solo per modalità cross-device*): ha lo scopo di invitare l’Utente a inquadrare il codice QR;  
- **waiting page** (*solo per modalità cross-device*): ha lo scopo di invitare l’Utente a continuare il processo di Autenticazione sulla propria Istanza IT-Wallet; 
```

- **thank you page**: ha lo scopo di comunicare all’Utente l'avvenuta Autenticazione; 
- **error page**: ha lo scopo di comunicare all’Utente eventuali errori legati al flusso di Autenticazione. 

> Tali pagine DEVONO prevedere elementi trasversali ricorrenti, in continuità con l’Identità Visiva del Touchpoint del Verificatore di Attestati Elettronici, in particolare: 

Reviewer: se usiamo DEVONO dobbiamo essere chiari su cosa, come i seguenti. Non possiamo usare DEVONO se non siamo chiari su cosa DEVONO

Reviewer 3: ```suggestion
Tali pagine DEVONO prevedere i seguenti elementi trasversali ricorrenti, in continuità con l’Identità Visiva del Touchpoint del Verificatore di Attestati Elettronici:
```

- un **header e/o un subheader**, che permette all'Utente di tornare alla pagina precedente; 
- un **footer** che include l’informativa privacy, le note legali e la Dichiarazione di Accessibilità, ove previsto da normativa. 

Di seguito invece gli elementi specifici caratteristici delle diverse pagine. 

**Discovery Page** 

Per garantire l’Autenticazione tramite il Sistema IT-Wallet, il Verificatore di Attestati Elettronici PUÒ aggiornare la propria Discovery Page con quella resa disponibile nelle Risorse Ufficiali. 

In alternativa, il Verificatore di Attestati Elettronici PUÒ mantenere la propria Discovery Page, ma DEVE in ogni caso integrare il Pulsante di Autenticazione, come da indicazioni presenti nella sezione :ref:`brand-identity:Pulsante di Autenticazione`.

In ogni caso: 

- la pagina DEVE presentare tutte le modalità di Autenticazione attraverso l’identità digitale tra cui la modalità di Autenticazione del Sistema IT-Wallet, quindi tramite il Pulsante di Autenticazione; 
- la pagina PUÒ presentare anche modalità di Autenticazione alternative, se disponibili; 

> - la pagina DOVREBBE garantire informazioni minime a supporto, per permettere all’Utente di compiere una scelta consapevole e informata. 

Reviewer: il linguaggio normativo non si applica alla pagina ma a chi la implementa, in questo caso il Verificatore di Attestati Elettronici DEVE implementare la pagina ...

Reviewer 3: ```suggestion
In ogni caso, nella pagina, il Verificatore di Attestati Elettronici:
- DEVE garantire la presenza di tutte le modalità di Autenticazione attraverso l’identità digitale tra cui la modalità di Autenticazione del Sistema IT-Wallet, quindi tramite il Pulsante di Autenticazione; 
- PUÒ presentare anche modalità di Autenticazione alternative, se disponibili; 
- DOVREBBE garantire informazioni minime a supporto, per permettere all’Utente di compiere una scelta consapevole e informata. 
```

Da modificare anche negli altri punti elenco di questa sezione (Autenticazione)

> Nel caso l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici da un Touchpoint diverso da quello su cui ha attivato l’Istanza IT-Wallet (modalità cross-dispositivo), la scelta di Autenticazione tramite il Sistema IT-Wallet DEVE condurre l’Utente alla QR code page. 

Reviewer: rivediamo i termini quali cross-dispositivo

Reviewer 3: ```suggestion
Nel caso l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici da un Touchpoint diverso da quello su cui ha attivato l’Istanza IT-Wallet (modalità cross-device), la scelta di Autenticazione tramite il Sistema IT-Wallet DEVE condurre l’Utente alla QR code page. 
```

> Nel caso in cui invece l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici dallo stesso Touchpoint su cui ha attivato l’Istanza IT-Wallet (modalità stesso dispositivo) tale pagina DEVE condurre l’Utente all’apertura della propria Istanza IT-Wallet. 

Reviewer: rivediamo stesso dispositivo

Reviewer 3: ```suggestion
Nel caso in cui invece l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici dallo stesso Touchpoint su cui ha attivato l’Istanza IT-Wallet (modalità same-device) tale pagina DEVE condurre l’Utente all’apertura della propria Istanza IT-Wallet. 
```

**QR code page (*solo per modalità cross-dispositivo*)** 

> La QR code page è la pagina su cui atterra l’Utente che ha scelto l’Autenticazione tramite il Sistema IT-Wallet in un flusso cross-dispositivo, e ha lo scopo di invitare l’Utente a scannerizzare, con la propria Istanza IT-Wallet, il codice QR generato. 

Reviewer: rivediamo cross dispositivo

Reviewer 3: ```suggestion
La QR code page è la pagina su cui atterra l’Utente che ha scelto l’Autenticazione tramite il Sistema IT-Wallet in un flusso cross-device, e ha lo scopo di invitare l’Utente a scannerizzare, con la propria Istanza IT-Wallet, il codice QR generato. 
```

Il Verificatore di Attestati Elettronici DOVREBBE implementare la QR code page resa disponibile nelle Risorse Ufficiali. In ogni caso: 

- la pagina DEVE includere gli elementi propri dell'Identità Visiva del Sistema IT-Wallet, tra cui il Logo; 
- la pagina DEVE esporre il codice QR e un testo sintetico e chiaro che inviti l'Utente a scannerizzarlo con la propria Istanza IT-Wallet; 
- la pagina DEVE includere un testo sintetico e chiaro che permetta all’Utente di conoscere la validità temporale del codice QR;  
- la pagina DEVE includere una Call to Action che, in caso di timeout, permetta all’Utente di generare un nuovo codice QR; 
- la pagina DEVE includere una Call to Action che permetta all’Utente di annullare l’operazione e tornare alla Discovery Page. 

Per garantire la leggibilità del codice QR: 

- si DEVENO rispettare le dimensioni minime raccomandate per garantire una scansione efficace. Una misura di 150x150 pixel è generalmente adeguata, ma per codici con alta densità di dati (e.g. URL lunghi o numerosi caratteri), è consigliabile aumentarla a 300x300 pixel o più; 

> - si DEVE mantenere un contrasto adeguato tra il codice QR e lo sfondo (la condizione ideale prevede uno sfondo bianco con un codice QR nero); 

Reviewer: qui bisogna normare i colori, il contrasto di conseguenza è sempre meglio bianco/nero

non lascerei la libertà agli implementatori di supporre soluzioni di contrasto presumibilmente adeguate

Reviewer 3: ```suggestion
Inoltre, nei rispetto di [RIF_ACCESSIBILITÁ], relativamente al codice QR, il Verificatore di Attestati Elettronici: 

- DEVE rispettare le dimensioni minime raccomandate per garantire una scansione efficace. Una misura di 150x150 pixel è generalmente adeguata, ma per codici con alta densità di dati (e.g. URL lunghi o numerosi caratteri), è consigliabile aumentarla a 300x300 pixel o più; 
- DEVE garantire un contrasto minimo tra il codice QR e lo sfondo (la condizione ideale prevede uno sfondo bianco con un codice QR nero); 
- DEVE evitare inversioni di colore tra sfondo e codice QR; 
- DEVE limitare la presenza a un solo codice QR per pagina; 
- DEVE garantire nitidezza e alta qualità; 
- DEVE garantire il formato SVG; 
- DEVE garantire che non venga parzialmente nascosto da testo o altri elementi.
```

- si DEVONO evitare inversioni di colore tra sfondo e codice QR; 
- si DEVE limitare la presenza a un solo codice QR per pagina; 

> - il codice QR DEVE essere nitido e di alta qualità (si consiglia il formato SVG); 

Reviewer: ```suggestion
- il codice QR DEVE essere nitido e di alta qualità;
- il codice QR DEVE essere in formato SVG; 
```

Reviewer 3: risolto (vedi commento sopra)

- non gli si DEVE sovrapporre testo o altri elementi che lo possano nascondere parzialmente. 

**Waiting page (*solo per modalità cross-dispositivo*)** 

La waiting page è la pagina che invita l’Utente a proseguire il processo di Autenticazione sulla propria Istanza IT-Wallet, a valle della scansione del codice QR. 

> Il Verificatore di Attestati Elettronici DOVREBBE implementare la waiting page (cross-dispositivo) resa disponibile nelle Risorse Ufficiali. In ogni caso: 

Reviewer: rivediamo cross dispositivo

Reviewer 3: ```suggestion
Il Verificatore di Attestati Elettronici DOVREBBE implementare la waiting page (cross-device) resa disponibile nelle Risorse Ufficiali. In ogni caso: 
```

- la pagina DEVE includere gli elementi propri dell'Identità Visiva del Sistema IT-Wallet, tra cui il Logo e un’icona o altro elemento grafico che aiuti a veicolare la natura dell'errore; 
- la pagina DEVE prevedere un testo sintetico e chiaro che spieghi all’Utente la natura dell’errore, il codice errore e una sua spiegazione semplice; 
- la pagina DEVE prevedere una o più Call to Action che invitino l’Utente a intraprendere le azioni previste (es. riprova, contatta l'assistenza, etc.). 

Gestione degli Attestati Elettronici
------------------------------------

Il Fornitore di Soluzione IT-Wallet, attraverso la propria Soluzione IT-Wallet, e il Fornitore di Dati di Identificazione Personale o il Fornitore di Attestati Elettronici di Attributi, attraverso Touchpoint dedicati, DEVONO dare all’Utente la possibilità di gestire in ogni momento i propri Attestati Elettronici.  

In questa sezione sono illustrate tre diverse categorie di requisiti per la gestione del ciclo di vita di ogni Attestato Elettronico e nello specifico per la gestione: 

- **del suo stato**: per consentire all’Utente di appurare la condizione di validità o invalidità di un Attestato Elettronico; 

> - **dei suoi utilizzi**: per consentire all’Utente di visualizzare e gestire lo storico delle transazioni effettuate con un Attestato Elettronico; 

Reviewer: ```suggestion
- **dei suoi utilizzi**: per consentire all’Utente di visualizzare e gestire lo storico delle transazioni effettuate utilizzando un Attestato Elettronico; 
```

- **dei suoi dati**: per consentire all’Utente di archiviare e ripristinare ogni Attestato Elettronico di Attributi in linea col principio di *data portability*. 

> Di seguito i principali aspetti che impattano e determinano l’Esperienza Utente nell’ambito della gestione degli Attestati Elettronici per mezzo di un’Istanza IT-Wallet e i requisiti funzionali riferiti a ciascuna categoria. 

Reviewer: qui bisogna aggiungere delle considerazioni di privacy

bisogna chiarire dove materialmente risiedano i dati dell'utente

Reviewer 3: se esiste già un contenuto di approfondimento, possiamo aggiungere un rimando

   - il documento fisico corrispondente all’Attestato Elettronico è stato dichiarato smarrito o danneggiato da parte dell’Utente tramite apposito canale/ Touchpoint; 
   - il documento fisico corrispondente all’Attestato Elettronico è stato revocato dalle autorità competenti; 
   - non sussistono più i requisiti minimi di sicurezza e/o affidabilità di una o più delle parti coinvolte; 
   - il dispositivo dell'Utente non soddisfa più i requisiti minimi di sicurezza (dispositivo rootato o jailbroken). 
 
Storico degli Attestati Elettronici 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Al fine di garantire i principi di visibilità e trasparenza, il Fornitore di Soluzione IT-Wallet DEVE permettere all’Utente di visualizzare lo storico di tutte le transazioni effettuate, ovvero l’utilizzo degli Attestati Elettronici di Attributi tramite l'Istanza IT-Wallet. In particolare: 

- l’Istanza IT-Wallet DEVE mostrare all’Utente con quali Verificatori di Attestati Elettronici ha interagito e quali Attestati Elettronici sono stati oggetto di presentazione e verifica; 

> - l’Istanza IT-Wallet DEVE permettere all’Utente di richiedere facilmente al Verificatore di Attestati Elettronici la cancellazione delle proprie informazioni oggetto delle precedenti presentazioni. 

Reviewer: non è in sovrapposizione con una sezione precedente dove ho commentato sulle considerazioni di privacy?

Reviewer 3: Nella sezione precedente vengono introdotte le successive sezioni di approfondimento, tra cui questa relativa allo Storico (al momento abbozzate in attesa di approfondimenti progettuali).

In caso di errori nell’utilizzo dell’Istanza IT-Wallet, il Fornitore di Soluzione IT-Wallet DEVE garantire all’Utente la visualizzazione di messaggi coerenti che lo informino e guidino alla loro risoluzione. Per approfondimenti si rimanda alla sezione :ref:`functionalities:Gestione degli errori`. 

Gestione degli errori 
---------------------

Il Sistema IT-Wallet prevede l’interazione di una molteplicità di servizi erogati da diversi attori. È quindi importante che venga definito un modello di gestione degli errori efficace, con l’obiettivo di migliorare la percezione e il senso di affidabilità dell’interno ecosistema e permettere all'Utente di sentirsi guidato nell'interazione con le diverse Soluzioni Tecniche e nella gestione consapevole di eventuali criticità durante la fruizione del servizio. 

Una comunicazione efficace in caso di errore determina un vantaggio anche per gli attori coinvolti, in quanto concorre alla riduzione delle richieste di assistenza e, quindi, alla minimizzazione dell’impatto sui sistemi.  

> Di seguito vengono presentati i requisiti e le principali buone pratiche di gestione dell’errore, riferite alle modalità di interazione tra l’Utente e la propria Istanza IT-Wallet. Tutti gli Attori Primari coinvolti nell'erogazione del servizio DEVONO inviare all’Utente, tramite l’Istanza IT-Wallet, input puntuali e granulari, ciascuno per i propri ambiti di competenza. Ciascun attore coinvolto DEVE, quindi, definire una lista di errori così da garantire al Fornitore di Soluzione IT-Wallet di approntare un modello efficace per la loro gestione all’interno della Soluzione IT-Wallet basato sulle seguenti dimensioni e variabili: 

Reviewer: ```suggestion
Di seguito vengono presentati i requisiti e le principali buone pratiche di gestione dell’errore, riferite alle modalità di interazione tra l’Utente e la propria Istanza IT-Wallet. Ciascun Attore Primario deve implementare una corretta gestione degli errori, in conformità alle attuali regole tecniche, al fine di comunicarli, direttamente o indirettamente, all’Utente e tramite l’Istanza IT-Wallet. Gli errori possono essere declinati, sulla base della loro natura, come segue: 
```

> Di seguito vengono presentati i requisiti e le principali buone pratiche di gestione dell’errore, riferite alle modalità di interazione tra l’Utente e la propria Istanza IT-Wallet. Tutti gli Attori Primari coinvolti nell'erogazione del servizio DEVONO inviare all’Utente, tramite l’Istanza IT-Wallet, input puntuali e granulari, ciascuno per i propri ambiti di competenza. Ciascun attore coinvolto DEVE, quindi, definire una lista di errori così da garantire al Fornitore di Soluzione IT-Wallet di approntare un modello efficace per la loro gestione all’interno della Soluzione IT-Wallet basato sulle seguenti dimensioni e variabili: 

Reviewer: non possiamo usare DEVE se non siamo precisi nell'indicare cosa di deve implementare

consiglio di rimandare nelle sezioni tecniche dove gli errori vengono definiti

Author: #729

- **Risoluzione autonoma**: permettere all’Utente di consultare domande frequenti (FAQ) sui contenuti e sulle funzionalità dell’Istanza IT-Wallet, al fine di risolvere eventuali casistiche di errore o problematiche in maniera autonoma. 

- **Apertura guidata di una segnalazione**: guidare l’Utente all’eventuale apertura di una segnalazione, in modo da circoscrivere la problematica e facilitare la sua gestione. 

- **Collaborazione tra attori**: rendere possibile un adeguato coordinamento tra tutti gli attori coinvolti (Fornitore di Soluzione IT-Wallet, Fornitore di Attestati Elettronici di Attributi, Fornitore di Dati di Identificazione Personale e Titolare di Fonte Autentica) in base al proprio ruolo e alle modalità operative specifiche. 

- **Comunicazione efficiente**: garantire all’Utente la possibilità di monitorare lo stato aggiornato della propria richiesta durante tutte le fasi di lavorazione, attraverso una comunicazione chiara, continua e coordinata.  

> Per applicare queste buone pratiche, gli attori coinvolti DOVREBBERO attuare i seguenti requisiti in modo da strutturare un modello di assistenza a livelli gerarchici: 

Reviewer: ```suggestion
Per applicare queste buone pratiche, gli attori coinvolti implementano i seguenti livelli di assistenza gerarchici: 
```

Reviewer 4: Non eliminerei il linguaggio normativo perché penso sia importante spingere gli attori a seguire il modello proposto. Potremmo riformulare così.
```suggestion
Per applicare queste buone pratiche, gli attori coinvolti DOVREBBERO implementare i seguenti livelli di assistenza gerarchici: 
```

	1. **I Livello – Gestione autonoma**: il Fornitore di Soluzione IT-Wallet DOVREBBE permettere all’Utente di disporre di una sezione di domande frequenti (FAQ) all’interno della propria Istanza IT-Wallet per chiarire dubbi e risolvere in autonomia alcune problematiche. Ogni attore DOVREBBE formulare delle domande frequenti e relative risposte specifiche rispetto a dati e funzionalità messe a disposizione al Fornitore di Soluzione IT-Wallet o nei propri Touchpoint. Per alcune casistiche di errore, il Fornitore di Soluzione IT-Wallet DOVREBBE rendere direttamente disponibile il canale di assistenza di un altro attore, per facilitare una gestione tempestiva ed evitare l’apertura di una richiesta di assistenza nell’Istanza IT-Wallet. 

> 	2. **II Livello – Richiesta di assistenza al Fornitore di Soluzione IT-Wallet**: se il I livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DEVE permettere all’Utente di aprire una o più richieste di assistenza.  Tali richieste DOVREBBERO essere gestite tramite l’Istanza IT-Wallet o altri Touchpoint del Fornitore di Soluzione IT-Wallet. Il Fornitore di Soluzione IT-Wallet DEVE effettuare una diagnosi e procedere alla risoluzione della problematica, se di sua competenza. 

Reviewer: questo deve è possibile solo se il livello 1 fosse implementato

tuttavia il livello 1 non normato come obbligatorio

fixare

Reviewer 4: sì, è corretto. Inserirei DOVREBBERO anche al punto 2

Reviewer 4: Suggerisco questa riformulazione:

```suggestion
	2. **II Livello – Richiesta di assistenza al Fornitore di Soluzione IT-Wallet**: se il I livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE permettere all’Utente di aprire una o più richieste di assistenza, effettuare una diagnosi e procedere alla risoluzione della problematica, se di sua competenza. Tali richieste DOVREBBERO essere gestite tramite l’Istanza IT-Wallet o altri Touchpoint del Fornitore di Soluzione IT-Wallet.
```

> 	3. **III Livello – Inoltro della richiesta all’attore responsabile della problematica**: se il II livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE garantire che la richiesta sia inoltrata all’attore responsabile (Fornitore di Attestati Elettronici di Attributi, Fornitore di Dati di Identificazione Personale e Titolare di Fonte Autentica) che DEVE farsi carico della risoluzione della problematica e comunicare l’esito all’Utente. 

Reviewer: ```suggestion
	3. **III Livello – Inoltro della richiesta all’attore responsabile della problematica**: se il II livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE garantire che la richiesta sia inoltrata all’attore responsabile (Fornitore di Attestati Elettronici di Attributi, Fornitore di Dati di Identificazione Personale e Titolare di Fonte Autentica) che si fa carico della risoluzione della problematica e comunicare l’esito all’Utente. 
```

> Di seguito i requisiti funzionali a supporto dell’Esperienza Utente relativi all’Assistenza che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 

Reviewer: anche questi, non sono requisiti funzionali

ridefinire lo scopo dei seguenti punti, rifrasando questo testo introduttivo

Reviewer 4: In coerenza con i nostri commenti alle sezioni precedenti suggerirei la seguente riformulazione:

```suggestion
Di seguito i requisiti di Esperienza Utente che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```
